webdriver-spec.html

```diff
@@ -3946,6 +3946,9 @@
  <li><p>Let <var>centre point</var> be a pair of <var>rect</var>’s
   (<a>rectangle x coordinate</a> + <var>horizontal centre offset</var>)
   and (<a>rectangle y coordinate</a> + <var>vertical centre offset</var>)
+  
+ <li><p>If <a>command</a> is intercepted by an <var>element</var> with a higher 
+  z-index, an <a>element intercepting click</a> <a>error</a> is returned.
```

Comment:
Stacking order is way more complicated than "compare z-index". A click could be "intercepted" by an element with lower z-index. I'm not sure what intercept means, could you define it more precisely? Is the "intercept" test based on DOM event handling or is it unrelated to existing hit testing/event dispatch mechanisms?